irohad/main/impl/pg_connection_init.cpp

```diff
@@ -280,7 +280,7 @@ CREATE TABLE IF NOT EXISTS account_has_grantable_permissions (
     PRIMARY KEY (permittee_account_id, account_id)
 );
 CREATE TABLE IF NOT EXISTS tx_positions (
-    creator_id varchar(64),
+    creator_id varchar(128),
```

Comment:
128 seems still not enough. as I can see, domain name is only (indirectly) limited by grpc limit of 4GB per message. sadly, we have to keep `text` data type or make a `foreign key`.